cockroachcloud/migrations-page.md

```diff
@@ -14,16 +14,28 @@ The **Migrations** page on the {{ site.data.products.db }} Console features a **
 - Create a new {{ site.data.products.serverless }} database that uses the converted schema. You specify the target database and database owner when [finalizing the schema](#finalize-the-schema). {% include cockroachcloud/migration/sct-self-hosted.md %}
 
 {{site.data.alerts.callout_info}}
-On the **Migrations** page, a *migration* refers to converting a schema for use with CockroachDB and creating a new database that uses the schema. It does not include moving data to the new database. For details on all steps required to complete a database migration, see [Migrate Your Database to CockroachDB](../{{version_prefix}}migration-overview.html).
+The **Migrations** page is used to convert a schema for use with CockroachDB and creating a new database that uses the schema. It does not include moving data to the new database. For details on all steps required to complete a database migration, see [Migrate Your Database to CockroachDB](../{{version_prefix}}migration-overview.html).
```

Comment:
Total nit: Maybe the following, so the structure in the sentence is parallel:

The Migrations page is used to convert a schema for use with CockroachDB and to create a new database that uses the schema.

@kathancox I accidentally deleted your comment! Sorry!